x-pack/plugins/actions/server/builtin_action_types/jira/schema.ts

```diff
@@ -40,7 +40,14 @@ export const ExecutorSubActionPushParamsSchema = schema.object({
     externalId: schema.nullable(schema.string()),
     issueType: schema.nullable(schema.string()),
     priority: schema.nullable(schema.string()),
-    labels: schema.nullable(schema.arrayOf(schema.string())),
+    labels: schema.nullable(
+      schema.arrayOf(
+        schema.string({
+          validate: (label) =>
```

Comment:
Did you try to use RegExp here for the validation?

Reply:
No 😄 . Do you want me to?

Reply:
It is not a blocker, but could be more extensible.

Reply:
Done!